puppet/zulip/manifests/postgres_common.pp

```diff
@@ -7,8 +7,6 @@
                         "ptop",
                         # Python modules used in our monitoring/worker threads
                         "python-gevent",
-                        "python-tz",
-                        "python-dateutil",
```

Comment:
(I guess alternatively we could require a virtualenv on dedicated postgres hosts)